### PR TITLE
[Workers] Pricing partials

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-8359

### CODE REVIEW

**Review by elithrar — APPROVED** [review]

File: content/workers/_partials/_queues_pricing.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+{{<table-wrap>}}
+
+|                     | Free Tier                    | Paid                       |
+| ------------------- | ---------------------------- | -------------------------- |
+| Standard operations | 1,000,000 operations / month | $0.40 / million operations |
+
+{{</table-wrap>}}
+
+In most cases, it takes 3 operations to deliver a message: 1 write, 1 read, and 1 delete. Therefore, you can use the following formula to estimate your monthly bill:
```
Comment (ranbel) [inline content/workers/_partials/_queues_pricing.md:30]: Should 3 and 1 be spelled out?

File: content/workers/_partials/_queues_pricing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    ((Number of Messages * 3) - 1,000,000) / 1,000,000  * $0.40
+
+Additionally:
+
+* Each retry incurs a read operation. A batch of 10 messages that is retried would incur 10 operations for each retry.
+* Messages that reach the maximum retries and that are written to a [Dead Letter Queue](/queues/learning/batching-retries) incur a write operation for each 64 KB chunk. A message that was retried 3 times (the default), fails delivery on the fourth time and is written to a Dead Letter Queue would incur five (5) read operations.
```
Comment (ranbel) [inline content/workers/_partials/_queues_pricing.md:37]:
```suggestion
* Messages that reach the maximum retries and that are written to a [Dead Letter Queue](/queues/learning/batching-retries) incur a write operation for each 64 KB chunk. A message that was retried three times (the default), fails delivery on the fourth time and is written to a Dead Letter Queue would incur five read operations.
```

File: content/workers/_partials/_transactional_storage_api_pricing.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+|                                  | Paid plan                  |
+| -------------------------------- | -------------------------- |
+| Read request units<sup>1,2</sup> | 1 million, + $0.20/million |
+| Write request units<sup>3</sup>  | 1 million, + $1.00/million |
+| Delete requests<sup>4</sup>      | 1 million, + $1.00/million |
+| Stored data<sup>5</sup>          | 1 GB, + $0.20/ GB-month    |
+
+{{</table-wrap>}}
+
+<sup>1</sup> A request unit is defined as 4 KB of data read or written. A request that writes or reads more than 4 KB will consume multiple units, for example, a 9 KB write will consume 3 write request units.
```
Comment (ranbel) [inline content/workers/_partials/_transactional_storage_api_pricing.md:23]:
```suggestion
<sup>1</sup> A request unit is defined as 4 KB of data read or written. A request that writes or reads more than 4 KB will consume multiple units, for example, a 9 KB write will consume three write request units.
```

File: content/workers/_partials/_transactional_storage_api_pricing.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+<sup>2</sup>  List operations are billed by read request units, based on the amount of data examined. For example, a list request that returns a combined 80 KB of keys and values will be billed 20 read request units. A list request that does not return anything is billed for 1 read request unit.
```
Comment (ranbel) [inline content/workers/_partials/_transactional_storage_api_pricing.md:25]:
```suggestion
<sup>2</sup>  List operations are billed by read request units, based on the amount of data examined. For example, a list request that returns a combined 80 KB of keys and values will be billed 20 read request units. A list request that does not return anything is billed as a single read request unit.
```

**Review by ranbel — APPROVED** [review]

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**